packages/story-editor/src/components/library/panes/shopping/shoppingPane.js

```diff
@@ -74,17 +75,27 @@ function ShoppingPane(props) {
     actions: { getProducts },
   } = useAPI();
 
+  const isShoppingEnabled =
+    'none' !== shoppingProvider && isShoppingIntegrationEnabled;
+
   const { currentPageProducts } = useStory(({ state: { currentPage } }) => ({
     currentPageProducts: currentPage?.elements
       ?.filter(({ type }) => type === 'product')
       .map(({ id, product }) => ({
         elementId: id,
-        productId: product?.productId,
+        product,
       })),
   }));
 
   const getProductsByQuery = useCallback(
     async (value = '') => {
+      if (!isShoppingEnabled) {
+        setProducts(currentPageProducts?.map(({ product }) => product));
+        setIsLoading(false);
```

Comment:
Wondering if this should be a standalone check outside of the Products query -- 

It might come in handy somewhere else in the code in the future.

Reply:
This logic has now been moved.